### ISSUE TITLE
Changes must able to tick/untick parent folder

### ISSUE THREAD

**Issue by DryreL** [desktop/desktop#14875]
### The feature request

Changes must able to tick/untick parent folder. For instance, we can select folders in GitKraken. I want same feature in the GitHub Desktop as well. It is very useful for me.

### Proposed solution

Sometimes it is necessary to batch files and I separate them into multiple commits thanks to selectable (tick-able) folders.

**Comment by tsvetilian-ty** [comment]
👋 @DryreL I'm not sure I understand your request 🤔 You can include the files into different commits by ticking or unticking them. You can use standard multi-select shortcuts such as `Shift + ArrowUp`,  `Shift + ArrowDown`, `Ctrl + Click` and use right-click to choose if you want to "Include/Exclude selected files"

### Example
![image](https://user-images.githubusercontent.com/9341546/175965183-6ca7fddd-a223-4f95-8e94-52e9eb86ecdc.png)

**Comment by DryreL** [comment]
Hi [tsvetilian-ty](https://github.com/tsvetilian-ty),

Yes I know **Shift** key but I want more efficient way by clicking folder name so it will choose all of the items in the folder instead of selecting every changes.

So when there are too many changes at once, I can more easily split it into multiple commits so that the changes are more understandable.

In the current situation I have to **hold down the Shift** key and **scroll** from the beginning of the folder to the end, which can be tiring when there are too many files.

**Comment by steveward** [comment]
Thanks for sharing that feedback @DryreL -- I'm going to add this to https://github.com/desktop/desktop/issues/4739 since this has been brought up there and is where we are tracking improvements for the Changes list.

**Comment by DryreL** [comment]
Hi @steveward 

Thank you. I'm waiting for this improvement.

**TL;DR:** I want to select some changes in the changes section based on the folder.

Here's an example below:

I have 1100+ changes but I just want to select **InfiniteBladeFireLands** folder's changes to commit.

![image](https://user-images.githubusercontent.com/20909077/176033743-0c3d4267-df01-4a45-96a9-a56a1cabb0e3.png)
